cluster/local/microk8s.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+set -e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# shellcheck disable=SC1090
+projectdir="${scriptdir}/../.."
+
+# get the build environment variables from the special build.vars target in the main makefile
+eval $(make --no-print-directory -C ${scriptdir}/../.. build.vars)
+
+MICROK8S_REGISTRY="localhost:32000"
+BUILD_IMAGE="${BUILD_REGISTRY}/${PROJECT_NAME}-amd64"
+# Use microk8s registry add-on
+MICROK8S_IMAGE="${MICROK8S_REGISTRY}/${PROJECT_NAME}:master"
+DEFAULT_NAMESPACE="crossplane-system"
+
+function copy_image_to_cluster() {
+    local build_image=$1
+    local final_image=$2
+    docker tag "${build_image}" "${final_image}" && docker push "${final_image}"
+    echo "Tagged image: ${final_image}"
+}
+
+# Deletes pods with application prefix. Namespace is expected as the first argument
+function delete_pods() {
+    for pod in $(kubectl get pods -n "$2" -l "app=$1" --no-headers -o custom-columns=NAME:.metadata.name); do
+        kubectl delete pod "$pod" -n "$2"
+    done
+}
+
+# current kubectl context == minikube, returns boolean
+function check_context() {
+    if [ "$(kubectl config view 2>/dev/null | awk '/current-context/ {print $NF}')" = "minikube" ]; then
```

Comment:
This is returning true (`0`) when "minikube" is the current context.
I would assume this should be `"microk8s"` so that the "restart" command works correctly.

In my local configuration, based on this branch, `kubectl config get-contexts` and `kubectl config get-clusters` doesn't show a "microk8s" value.. The `awk` expression above results in `""` for me.